packages/compiler/core/binder.ts

```diff
@@ -389,6 +389,7 @@ export function createBinder(program: Program, options: BinderOptions = {}): Bin
   function bindOperationStatement(statement: OperationStatementNode) {
     if (scope.kind !== SyntaxKind.InterfaceStatement) {
       declareSymbol(statement, SymbolFlags.Operation);
+      statement.locals = new SymbolTable();
```

Comment:
is this actually used?

Reply:
It's where the template parameters get bound. This happens when we bind a template parameter declaration and we call declareSymbol which, unless you're declaring in a namespace or a script container, adds the symbol to the locals.

Reply:
oh right, makes sense, forgot about template parameters